modules/administration/pages/tshoot-sync.adoc

```diff
@@ -47,7 +47,7 @@ For more on importing GPG keys, see xref:administration:repo-metadata.adoc[].
 
 
 GPG Key Removal from [literal]``spacewalk-repo-sync``::
-Once a GPG key for repository has been manually imported using [literal]``spacewalk-repo-sync`` command, and this key is no longer needed (for example if the the key was  compromised, or was used for testing purposes only), it can be removed from the zypper RPM database used by [literal]``spacewalk-repo-sync`` using the following command:
+When a GPG key for repository has been manually imported using [literal]``spacewalk-repo-sync`` command, and this key is no longer needed (for example if the the key was  compromised, or was used for testing purposes only), it can be removed from the zypper RPM database used by [literal]``spacewalk-repo-sync`` using the following command:
```

Comment:
```suggestion
When a GPG key for repository has been manually imported using [literal]``spacewalk-repo-sync`` command, and this key is no longer needed (for example if the the key was compromised, or was used for testing purposes only), it can be removed from the zypper RPM database used by [literal]``spacewalk-repo-sync`` using the following command:
```